site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -130,12 +169,34 @@ chrome.runtime.onInstalled.addListener(details => {
 });
 ```
 
+## Extension examples {: #more-samples }
+
+See the [Manifest V3 - Web Accessible Resources demo][github-war-sample] for more Runtime API examples.
```

Comment:
This is the only example I found in MV3. 

I looked at the [MV2 messaging API example](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/mv2-archive/api/messaging/timer), but couldn't get it to work. I'd like to create a very easy-to-understand extension example that explains how message passing works. I will do this when I work on the messages CUJ.

Reply:
Sounds good 👍🏻